ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_security.schema.yml

```diff
@@ -21,6 +21,55 @@ keys:
             type: bool
           encryption_reversible:
             type: str
+          policies:
+            type: list
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                minimum:
+                  type: dict
+                  keys:
+                    digits:
+                      type: int
+                      min: 0
+                      convert_types:
+                      - str
+                    length:
+                      type: int
+                      min: 0
+                      convert_types:
+                      - str
+                    lower:
+                      type: int
+                      min: 0
+                      convert_types:
+                      - str
+                    special:
+                      type: int
+                      min: 0
+                      convert_types:
+                      - str
+                    upper:
+                      type: int
+                      min: 0
+                      convert_types:
+                      - str
+                maximum:
```

Comment:
Minimum values are 1  for maximum limits according to EOS. Also please set maximum values to 65535 everywhere